### DISCUSSION TITLE
Roadmap of planned features or fixes?

### DISCUSSION THREAD

**Discussion by hermasj** [axios/axios#5273] [General]
Do you make available any sort of "roadmap" of planned upcoming features or fixes?  Short term or long term... "in the works" or even just tentatively planned?

I have seen some release notes documents here that I assume were published following a release, but wonder if something similar might be available ahead of time for upcoming releases.

**Comment by jasonsaayman** [comment]
not really, we have added a pinned issue for a sort of roadmap
